pyiron/lammps/interactive.py

```diff
@@ -563,23 +563,16 @@ def interactive_stress_getter(self):
             self._interactive_lib_command("compute st all stress/atom NULL")
             self._interactive_lib_command("run 0")
             self.interactive_cache["stress"] = []
-        ss = (
-            np.array(
-                [
-                    self._interactive_library.extract_compute("st", 1, 2)[i][
-                        j + (j != k) * (k + 2)
-                    ]
-                    for i in range(len(self.structure))
-                    for j in range(3)
-                    for k in range(3)
-                ]
-            ).reshape(len(self.structure), 3, 3)
-            / 1.602e6
-        )
+        ID = self._interactive_library.extract_atom("id", 0)
```

Comment:
`id` is a python function and writing it in capital letters is even less pythonic - can not we use something like `id_lst` or `atom_ids` ?